web/src/containers/ActivityDetailContractorResources.js

```diff
@@ -215,11 +240,80 @@ class ContractorExpenses extends Component {
                         label={year}
                         value={contractor.years[year]}
                         onChange={this.handleYearChange(i, year)}
+                        disabled={contractor.hourly.useHourly}
                         wrapperClass="mr2 flex-auto"
                       />
                     ))}
                   </div>
                 </div>
+                <div className="mb3 md-flex">
+                  <Label>
+                    {t('activities.contractorResources.labels.hourly')}
+                  </Label>
+                  <div className="md-col-6">
+                    <div className="flex items-center">
+                      <div className="mr2">This is an hourly resource</div>
```

Comment:
Resource text.  I'm comfortable just making a note of this and fixing it later.  Can be part of our catch-up sprint.

Reply:
👍